cookbook/job_execution.rst

```diff
@@ -0,0 +1,56 @@
+How to Purge jobs executions
+============================
+
+This chapter gives details on how to purge jobs executions in order to decrease the database volume.
+
+The batch job command
+---------------------
+
+Akeneo PIM provides a simple command to purge jobs executions:
+
+.. code-block:: bash
+    :linenos:
+
+    app/console akeneo:batch:purge-job-execution [-d|--days DAYS]
```

Comment:
As you explained the option “--days” just below, I am not sure it is relevant to put it here